realsense_navigation/package.xml

```diff
@@ -1,23 +1,27 @@
 <?xml version="1.0"?>
-<package>
+<package format="2">
   <name>realsense_navigation</name>
   <version>1.1.0</version>
   <description>RealSense Camera navigation package allowing use of Intel 3D cameras with the navigation stack</description>
 
-  <maintainer email="amit.moran@intel.com">Amit Moran</maintainer>
   <maintainer email="matthew.c.curfman@intel.com">Matt Curfman</maintainer>
+  <maintainer email="mark.d.horn@intel.com">Mark Horn</maintainer>
 
   <license>BSD 3-clause. See license attached</license>
 
   <url type="website">http://www.ros.org/wiki/RealSense</url>
 
   <author email="amit.moran@intel.com">Amit Moran</author>
-  <author email="matthew.c.curfman@intel.com">Matt Curfman</author>
 
   <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>roscpp</build_depend>
-  <build_depend>rospy</build_depend>
- 
-  <run_depend>roscpp</run_depend>
-  <run_depend>rospy</run_depend>
+
+  <exec_depend>roslaunch</exec_depend>
+  <exec_depend>realsense_camera</exec_depend>
```

Comment:
@kevincwells Considering we have the realsense_navigation package in the same repository and we know it will indirectly require the realsense_camera package, are you opposed to leaving that dependency?